CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ This file is used to list changes made in each version of the aws-parallelcluste
 ------
 
 **ENHANCEMENTS**
+- Add a clustermgtd config option `ec2_instance_missing_max_count` to allow a configurable amount of time for eventual EC2 
```

Comment:
MINOR - instead of "to allow a configurable amount of time" what about "to allow a configurable amount of retry"?

Reply:
that's fair, more correct